# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvidia/module-install-24.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- NVIDIA-Linux-x86_64-100.14.19-pkg2/usr/src/nv/Makefile.nvidia.orig	2007-09-12 14:53:20.000000000 -0700
+++ NVIDIA-Linux-x86_64-100.14.19-pkg2/usr/src/nv/Makefile.nvidia	2007-10-18 15:51:53.000000000 -0700
@@ -130,9 +130,7 @@
 	mkdir -p $(MODULE_ROOT)/video && \
 	install -m 0664 -o root -g root $(MODULE_NAME).o \
 	 $(MODULE_ROOT)/video/$(MODULE_NAME).o && \
-	PATH="$(PATH):/bin:/sbin" depmod -a && \
-		PATH="$(PATH):/bin:/sbin" modprobe $(MODULE_NAME) && \
-	echo "$(MODULE_NAME).o installed successfully."; \
+	echo "$(MODULE_NAME).o installed successfully."
 
 $(VERSION_HDR): 
 	echo \#define NV_COMPILER \"`$(CC) -v 2>&1 | tail -n 1`\" > $@
